cdci_data_analysis/configurer.py

```diff
@@ -46,6 +46,9 @@ def __init__(self, required_keys=None, allowed_keys=None, **kwargs):
         if allowed_keys is None:
             #temporary hardcode smth to preserve interface
             allowed_optional_keys = ['data_server_cache']
+
+            # preserving interface requires listing these keys, currently provided by all plugins (to be updated)
+            allowed_optional_keys += ['data_server_remote_cache', 'dispatcher_mnt_point']
```

Comment:
These keys are implicitly allowed in process_integral_keys
So this will change nothing in the behaviour, but at least to have it declared explicitly is good for clarity.

Reply:
but it does change the interface!

if the key is None, since None is handed in process_integral_keys the same way as unset key, and then it fails because it is not allowed.
and None is routinely provided by plugins, apparently.

it could be handled by replacing get with pop, but this means that yet another argument of process_integral_keys will be mutating, leading for possible further confusion.

Reply:
I show it with the updated test below.